src/vtbackend/Functions.h

```diff
@@ -618,10 +623,82 @@ inline auto const& functions() noexcept
     return funcs;
 }
 
+class SupportedSequences
+{
+  public:
+    SupportedSequences()
+    {
+        for (auto [index, seq]: crispy::indexed(functions()))
+        {
+            _supportedSequences[index] = seq;
+        }
+        crispy::sort(
+            _supportedSequences,
+            [](FunctionDefinition const& a, FunctionDefinition const& b) constexpr { return compare(a, b); });
+        _lastIndex = _supportedSequences.size();
+    }
+
+    [[nodiscard]] gsl::span<FunctionDefinition const> getAllSequences() const noexcept
+    {
+        return gsl::span<FunctionDefinition const>{_supportedSequences.cbegin(), _supportedSequences.cend()};
+    }
+
+    [[nodiscard]] gsl::span<FunctionDefinition const> getSupportedSequencesAtCurrentVTLevel() const noexcept
```

Comment:
```suggestion
    [[nodiscard]] gsl::span<FunctionDefinition const> activeSequences() const noexcept
```

I think, since this class is all about restricting the set of available VT sequences based on VT level, we do not need to name this member like that but simply give it a name that denotes what you're accessing, an array of (active) sequences. We could also get rid of the `get...` prefixes in both functions.